src/services/fileStorage/proxy-service.js

```diff
@@ -92,7 +91,7 @@ const prepareThumbnailGeneration = (file, strategy, userId, { name: dataName },
  * @returns {Promise} Promise that rejects with errors or resolves with no data otherwise
  */
 const prepareSecurityCheck = (file, userId, strategy) => {
-	if (ENABLE_FILE_SECURITY_CHECK === 'true') {
+	if (Configuration.get('ENABLE_FILE_SECURITY_CHECK')) {
```

Comment:
```suggestion
	if (Configuration.get('ENABLE_FILE_SECURITY_CHECK') === true) {
```
avoid implicit conditions